pkg/rancher-desktop/pages/containers/logs/_id.vue

```diff
@@ -223,14 +231,27 @@ export default defineComponent({
               if (this.terminal && (data.stdout || data.stderr)) {
                 const output = data.stdout || data.stderr;
 
-                const buffer = this.terminal.buffer.active;
-                const viewport = this.terminal.rows;
-                const isAtBottom = buffer.viewportY >= buffer.length - viewport;
+                this.hasReceivedLogs = true;
 
                 this.terminal.write(output);
 
-                if (isAtBottom) {
-                  this.terminal.scrollToBottom();
+                if (this.waitingForInitialLogs) {
+                  if (this.revealTimeout) {
+                    clearTimeout(this.revealTimeout);
+                  }
```

Comment:
So if we receive more logs before the timeout expires, we delay the timeout some more? (Nothing wrong with that, just confirming intended behaviour.)